prometheus-serde/main.go

```diff
@@ -29,11 +28,6 @@ func processPrometheusData(req *prompb.WriteRequest) ([][]byte, error) {
 		for _, sample := range ts.Samples {
 			name := labels["__name__"]
 
-			if math.IsNaN(sample.Value) {
```

Comment:
won't your `strconv.FormatFloat` fail?

Reply:
I checked it won't fail for NaN, +Inf, -Inf